### PR TITLE
Add new fields for arbitration history

### PR DESCRIPTION
**Pull request by EddieOrbit**
## Description

- Source `arbitration.history`
- Field with path `arbitration.items[].url` fillable by `arbitration.history`
- Field with path `arbitration.items[].type` fillable by `arbitration.history`
- Field with path `arbitration.items[].number` fillable by `arbitration.history`
- Field with path `arbitration.items[].court.name` fillable by `arbitration.history`
- Field with path `arbitration.items[].description` fillable by `arbitration.history`
- Field with path `arbitration.items[].date.registration` fillable by `arbitration.history`

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I wrote unit tests for my code _(if tests is required for my changes)_
- [X] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: sources/default/sources_list.json [hunk 1]
```diff
@@ -223,5 +223,10 @@
         "name": "insurance.dtp",
         "description": "Информация о ДТП из РСА (европротоколы)",
         "enabled": true
+    },
+    {
+        "name": "arbitration.history",
+        "description": "История о судебных делах в которых фигурировал ТС",
```
Comment (Zaboday) [inline sources/default/sources_list.json:229]: `История судебных дел с участием ТС`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## UNRELEASED
+
+### Added
+
+- Source `arbitration.history`
```
Comment (Zaboday) [inline CHANGELOG.md:11]: Убери `fillable by`

```
- Source `arbitration.history`
- Field with path `arbitration.items[].url`
- Field with path `arbitration.items[].type`
- Field with path `arbitration.items[].number`
- Field with path `arbitration.items[].court.name`
- Field with path `arbitration.items[].description`
- Field with path `arbitration.items[].date.registration`
```

File: sources/default/sources_list.json (hunk 1 above)
Comment (EddieOrbit) [reply sources/default/sources_list.json]: Готово

File: CHANGELOG.md (hunk 2 above)
Comment (EddieOrbit) [reply CHANGELOG.md:11]: Готово

**Pull request merged**